src/components/App.js

```diff
@@ -1,15 +1,33 @@
-import React from 'react';
+import React, { Component } from 'react';
 import '../index.css';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] We can safely remove this eslint disable line since you do not have any unused vars here.